crates/standalone/src/lib.rs

```diff
@@ -438,13 +439,12 @@ fn withdraw_energy(control_db: &ControlDb, identity: &Identity, amount: EnergyQu
 pub async fn exec_subcommand(cmd: &str, args: &ArgMatches) -> Result<(), anyhow::Error> {
     match cmd {
         "start" => start::exec(args).await,
-        "version" => version::exec(args).await,
```

Comment:
reviewer note: this is now provided by `--version` in `crates/standalone/src/main.rs`